### PR TITLE
new feature licenses create-options

### PR DESCRIPTION
**Pull request by caberos**
https://github.com/softlayer/softlayer-python/issues/1507

### CODE REVIEW

File: SoftLayer/CLI/licenses/create_options.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+from SoftLayer.managers import licenses
+from SoftLayer import utils
+
+
+@click.command()
+@environment.pass_env
+def cli(env):
+    """Server order options for a given chassis."""
+
+    licenses_manager = licenses.LicensesManager(env.client)
```
Comment (ATGE) [inline SoftLayer/CLI/licenses/create_options.py:17]:
```suggestion
    licenses_manager = LicensesManager(env.client)
```

File: SoftLayer/CLI/licenses/create_options.py [hunk 2]
```diff
@@ -0,0 +1,28 @@
+"""Licenses order options for a given VMware licenses."""
+# :license: MIT, see LICENSE for more details.
+
+import click
+
+from SoftLayer.CLI import environment
+from SoftLayer.CLI import formatting
+from SoftLayer.managers import licenses
```
Comment (ATGE) [inline SoftLayer/CLI/licenses/create_options.py:8]:
```suggestion
from SoftLayer.managers.licenses import LicensesManager
```

**Review by ATGE — COMMENTED** [review]
Currently looks like just displays the VMware license options, so I'm not sure if should be displayed Control Pane licenses too.

Also, there is a new manager module too: `LicensesManager`, please, add a unit test for that.

**Review by FernandoOjeda — COMMENTED** [review]
Do not forget to add unit test for the `licenses` managers module.

**Review by allmightyspiff — APPROVED** [review]
looks good, thanks

**Comment by allmightyspiff** [comment]
@caberos Looks like there are some conflicts with your other branch, would your mind resolving those, thanks.

**Pull request merged**